multimodal/src/autogluon/multimodal/models/huggingface_text.py

```diff
@@ -28,6 +28,15 @@
 logger = logging.getLogger(AUTOMM)
 
 
+class DummyLayer(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.dummy_bias = nn.Parameter(torch.ones(1, dtype=torch.float32))
```

Comment:
Do we need to specify `dtype=torch.float32`? I think lightning can automatically handle this.

Reply:
Lightning will set the dtype/device of the input to forward. For parameters, their dtype will be converted if we call functions like mod.half().